subsys/logging/log_backend_spinel.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log_backend.h>
+#include <logging/log_output.h>
+#include "log_backend_std.h"
+
+#ifndef CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE
+#define CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE 0
+#endif
+
+static u8_t char_buf[CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE];
+static bool panic_mode;
+
+static int write(u8_t *data, size_t length, void *ctx);
+
+LOG_OUTPUT_DEFINE(log_output, write,
+		  char_buf, sizeof(char_buf));
+
+static void put(const struct log_backend *const backend,
+		struct log_msg *msg)
+{
+	// prevent adding CRLF, which may crash spinel decoding
```

Comment:
run the checkpatch script on the files. I'm not sure c++ comments are allowed

Reply:
I did and there were no errors, but as Łukasz Maciejończyk wrote it is not allowed, so was fixed.